website/project/views/register.py

```diff
@@ -115,6 +339,8 @@ def node_register_template_page(auth, node, **kwargs):
 @must_have_permission(ADMIN)
 @must_not_be_registration
 def project_before_register(auth, node, **kwargs):
+    """Returns prompt informing user that addons, if any, won't be registered."""
```

Comment:
This behavior might change in the near future (Archiver). Comment is probably harmless for now, but feels pretty vulnerable to rot.